usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -128,6 +134,9 @@ def construct_idv_response(requested_award_dict: dict) -> OrderedDict:
         return None
     response.update(award)
 
+    account_data = fetch_account_details_award(award["id"])
```

Comment:
Is this supposed to be `fetch_account_details_idv`?

Reply:
:o yeah

Reply:
I believe the code is correct after the changes since the comments. Is that correct?

Reply:
yes